scm/driver/gitlab/webhook.go

```diff
@@ -601,21 +601,19 @@ type (
 			Homepage    string `json:"homepage"`
 		} `json:"repository"`
 		MergeRequest struct {
-			AssigneeID     interface{} `json:"assignee_id"`
-			AuthorID       int         `json:"author_id"`
-			CreatedAt      string      `json:"created_at"`
-			DeletedAt      interface{} `json:"deleted_at"`
-			Description    string      `json:"description"`
-			HeadPipelineID interface{} `json:"head_pipeline_id"`
-			ID             int         `json:"id"`
-			Iid            int         `json:"iid"`
-			LastEditedAt   interface{} `json:"last_edited_at"`
-			LastEditedByID interface{} `json:"last_edited_by_id"`
-			MergeCommitSha string      `json:"merge_commit_sha"`
-			MergeError     interface{} `json:"merge_error"`
-			MergeParams    struct {
-				ForceRemoveSourceBranch string `json:"force_remove_source_branch"`
-			} `json:"merge_params"`
+			AssigneeID                interface{} `json:"assignee_id"`
+			AuthorID                  int         `json:"author_id"`
+			CreatedAt                 string      `json:"created_at"`
+			DeletedAt                 interface{} `json:"deleted_at"`
+			Description               string      `json:"description"`
+			HeadPipelineID            interface{} `json:"head_pipeline_id"`
+			ID                        int         `json:"id"`
+			Iid                       int         `json:"iid"`
+			LastEditedAt              interface{} `json:"last_edited_at"`
+			LastEditedByID            interface{} `json:"last_edited_by_id"`
+			MergeCommitSha            string      `json:"merge_commit_sha"`
+			MergeError                interface{} `json:"merge_error"`
+			MergeParams               interface{} `json:"-"`
```

Comment:
Why is this removed? Should it be changed instead to 
```
ForceRemoveSourceBranch bool `json:"force_remove_source_branch"`
```

and verified that gitlab sends boolean instead of string (also fix testdata)?

Reply:
[force_remove_source_branch type is inconsistent](https://gitlab.com/gitlab-org/gitlab/-/issues/15647)